packages/tizen_rpc_port/tizen/src/tizen_rpc_port_plugin.cc

```diff
@@ -0,0 +1,856 @@
+#include "tizen_rpc_port_plugin.h"
+
+// For getPlatformVersion; remove unless needed for your plugin implementation.
+#include <flutter/event_channel.h>
+#include <flutter/event_sink.h>
+#include <flutter/event_stream_handler_functions.h>
+#include <flutter/method_channel.h>
+#include <flutter/plugin_registrar.h>
+#include <flutter/standard_message_codec.h>
+#include <flutter/standard_method_codec.h>
+#include <system_info.h>
+
+#include <functional>
+#include <memory>
+#include <sstream>
+#include <string>
+#include <unordered_map>
+
+#include "log.h"
+#include "rpc_port_proxy.h"
+#include "rpc_port_stub.h"
+
+namespace {
+using namespace tizen;
+
+class TizenRpcPortPlugin : public flutter::Plugin {
+ public:
+  static void RegisterWithRegistrar(flutter::PluginRegistrar* registrar) {
+    auto channel =
+        std::make_unique<flutter::MethodChannel<flutter::EncodableValue>>(
+            registrar->messenger(), "tizen/rpc_port",
+            &flutter::StandardMethodCodec::GetInstance());
+
+    auto plugin = std::make_unique<TizenRpcPortPlugin>(registrar);
+
+    channel->SetMethodCallHandler(
+        [plugin_pointer = plugin.get()](const auto& call, auto result) {
+          plugin_pointer->HandleMethodCall(call, std::move(result));
+        });
+
+    registrar->AddPlugin(std::move(plugin));
+  }
+
+  explicit TizenRpcPortPlugin(flutter::PluginRegistrar* plugin_registrar)
+      : plugin_registrar_(plugin_registrar) {
+    method_handlers_ = {
+        {"stubCreate", std::bind(&TizenRpcPortPlugin::StubCreate, this,
+                                 std::placeholders::_1, std::placeholders::_2)},
+        {"stubDestroy",
+         std::bind(&TizenRpcPortPlugin::StubDestroy, this,
+                   std::placeholders::_1, std::placeholders::_2)},
+        {"stubSetTrusted",
+         std::bind(&TizenRpcPortPlugin::StubSetTrusted, this,
+                   std::placeholders::_1, std::placeholders::_2)},
+        {"stubAddPrivilege",
+         std::bind(&TizenRpcPortPlugin::StubAddPrivilege, this,
+                   std::placeholders::_1, std::placeholders::_2)},
+        {"stubListen", std::bind(&TizenRpcPortPlugin::StubListen, this,
+                                 std::placeholders::_1, std::placeholders::_2)},
+        {"portSend", std::bind(&TizenRpcPortPlugin::PortSend, this,
+                               std::placeholders::_1, std::placeholders::_2)},
+        {"portReceive",
+         std::bind(&TizenRpcPortPlugin::PortReceive, this,
+                   std::placeholders::_1, std::placeholders::_2)},
+        {"proxyConnect",
+         std::bind(&TizenRpcPortPlugin::ProxyConnect, this,
+                   std::placeholders::_1, std::placeholders::_2)},
+        {"proxyProxyConnectSync",
```

Comment:
```suggestion
        {"proxyConnectSync",
```

Reply:
> 

Natives API can create multiple proxies.
But the plugin not supported by current implementations. I will add it in the next patch.